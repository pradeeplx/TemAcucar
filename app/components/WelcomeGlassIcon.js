import React from 'react'
import { Image, PixelRatio, Dimensions } from 'react-native'

export default WelcomeGlassIcon = ({ name, style }) => {
  const ratio = PixelRatio.get()
  let image
  if (ratio <= 1) {
    image = require('../img/glass100.png')
  } else if (ratio <= 1.5) {
    image = require('../img/glass150.png')
  } else if (ratio <= 2) {
    image = require('../img/glass200.png')
  } else if (ratio <= 3) {
    image = require('../img/glass300.png')
  } else if (ratio <= 3.5) {
    image = require('../img/glass350.png')
  } else {
    image = require('../img/glass400.png')
  }
  return (
    <Image source={image} style={[{
      width: 124,
      height: 124,
    }, style]}/>
  )
}
