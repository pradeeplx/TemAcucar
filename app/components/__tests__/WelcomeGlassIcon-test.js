import 'react-native'
import React from 'react'
import WelcomeGlassIcon from '../WelcomeGlassIcon'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <WelcomeGlassIcon style={{top: 0}}>FooBar</WelcomeGlassIcon>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
