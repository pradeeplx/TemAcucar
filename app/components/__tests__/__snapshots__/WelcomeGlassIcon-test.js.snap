exports[`test renders correctly 1`] = `
<Image
  source={1}
  style={
    Array [
      Object {
        "height": 124,
        "width": 124,
      },
      Object {
        "top": 0,
      },
    ]
  } />
`;
